tests/markFeatureWriter_test.py

```diff
@@ -16,9 +16,10 @@ def test_add_classes(self):
         glyph.appendAnchor(glyph.anchorClass(
             anchorDict={'name': '_bottom', 'x': 100, 'y': 0}))
         lines = []
-        writer = MarkFeatureWriter(
-            ufo, anchorList=(('bottom', '_bottom'),), mkmkAnchorList=(),
-            ligaAnchorList=((('top_1', 'top_2'), '_top'),))
+        writer = MarkFeatureWriter(ufo)
+        writer.anchorList = (('bottom', '_bottom'),)
+        writer.mkmkAnchorList = ()
+        writer.ligaAnchorList = ((('top_1', 'top_2'), '_top'),)
```

Comment:
Well, looking at the code we're commenting on, you would need to subclass and override setupAnchorPairs to set your own anchors instead of running the inference routine.